x-pack/plugins/security_solution/public/common/components/link_to/redirect_to_users.tsx

```diff
@@ -9,10 +9,10 @@ import type { UsersTableType } from '../../../users/store/model';
 import { appendSearch } from './helpers';
 
 export const getUsersDetailsUrl = (detailName: string, search?: string) =>
-  `/${detailName}${appendSearch(search)}`;
+  `name/${detailName}${appendSearch(search)}`;
```

Comment:
Do you also need the leading `/` here?

Reply:
Yes, I missed it. It seems to work without the `/`, but it is better to keep the URL consistent. Thanks!